Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -1099,6 +1100,13 @@ checkPaths pkg =
   | pat <- extraDocFiles pkg
   , Left err <- [parseFileGlob (specVersion pkg) pat]
   ]
+  ++
+  [ PackageBuildWarning $
+        "Scripts do not support the field: " ++ field ++ " "
+     ++ "If you need this field, please use a full package."
```

Comment:
minor thaing about the output: the upper case without a previous period look a bit strange:

```
Warning: Scripts do not support the field: include-dirs If you need this
field, please use a full package.
```
```suggestion
        "Scripts do not support the field: '" ++ field ++ "'. "
     ++ "If you need it, please use a full package."
```